Jenkinsfile

```diff
@@ -22,7 +22,7 @@ pipeline {
             steps {
                 sh '''#!/bin/bash -ex
                     source scenario/bin/activate
-                    flake8 piqe_ocp_lib/* --show-source --max-line-length=120 --ignore E731
+                    flake8 piqe_ocp_lib/*
```

Comment:
We could use poetry here but before we need to refactor the entire pipeline to use poetry instead of virtualenv. Left for a further issue.